config/test.exs

```diff
@@ -10,5 +10,8 @@ config :erlef, :env, :test
 
 config :erlef, Erlef.Mailer, adapter: Swoosh.Adapters.Test
 
+config :erlef, :wild_apricot_base_api_url, "http://127.0.0.1:9999"
+config :erlef, :wild_apricot_base_auth_url, "http://127.0.0.1:9999"
```

Comment:
In line with other things that we decided to pull in from system env, I would like to pull these in from there as well, but it makes testing more complicated.